lib/internal/http2/core.js

```diff
@@ -1906,7 +1907,8 @@ class Http2Stream extends Duplex {
       rstCode: NGHTTP2_NO_ERROR,
       writeQueueSize: 0,
       trailersReady: false,
-      endAfterHeaders: false
+      endAfterHeaders: false,
+      bytesWritten: 0
```

Comment:
nit: add a trailing comma so future changes are less noisy
```suggestion
      bytesWritten: 0,
```